### ISSUE TITLE
crank is not compatible with tomcat 6

### ISSUE THREAD

**Issue by GoogleCodeExporter** [bestlong/krank#23]
```
Because of this bug in tomcat

http://issues.apache.org/bugzilla/show_bug.cgi?id=42866

It is very picky to make crank works reliably under tomcat 6(at least 
including 6.0.14).

I have to follow the suggestion to change to server side state saving and 
MUST use facelets 1.1.14(1.1.13 would give NPE) 

Just hope to let others know if they encountered the same issue as it 
takes me for quite a while to test all the combination to close in to the 
tomcat bug and search for the bug database.

In summary, to deploy to tomcat 6(at least including 6.0.14):

facelets 1.1.14 is needed
must use server side state saving

until the tomcat bug is fixed or apply the mentioned fix to jasper-el.

```

Original issue reported on code.google.com by `garyng2...@gmail.com` on 10 Jan 2008 at 3:15
